Cargo.toml

```diff
@@ -45,7 +45,7 @@ write = ["write_core", "coff", "elf", "macho"]
 
 # Enable things that require libstd.
 # Currently, this provides an `Error` implementation.
-std = ["memchr/std"]
+std = ["memchr/std", "indexmap/std"]
```

Comment:
Will this enable the `indexmap` dependency when `std` is enabled but `write_core` is disabled? It might be more correct to make this a dependency of `write_core`.

Reply:
Good point; I've now moved it to `write_core`.